easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,422 @@
+    }),
+    ('psutil', '5.9.0', {
+        'checksums': ['869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25'],
+    }),
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('singledispatch', '3.7.0', {
+        'checksums': ['c1a4d5c1da310c3fd8fccfb8d4e1cb7df076148fd5d858a819e37fffe44f3092'],
+    }),
+    ('sortedcontainers', '2.4.0', {
+        'checksums': ['25caa5a06cc30b6b83d11423433f65d1f9d76c4c6a0c90e3379eaa43b9bfdb88'],
+    }),
+    ('intervaltree', '3.1.0', {
+        'checksums': ['902b1b88936918f9b2a19e0e5eb7ccb430ae45cde4f39ea4b36932920d33952d'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'checksums': ['f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d'],
+    }),
+    ('futures', '3.3.0', {
+        'modulename': False,
+        'checksums': ['7e033af76a5e35f58e56da7a91e687706faf4e7bdfb2cbc3f2cca6b9bcda9794'],
+    }),
+    ('glob2', '0.6', {
+        'checksums': ['f5b0a686ff21f820c4d3f0c4edd216704cea59d79d00fa337e244a2f2ff83ed6'],
+    }),
+    ('subprocess32', '3.5.4', {
+        'checksums': ['eb2937c80497978d181efa1b839ec2d9622cf9600a039a79d0e108d1f9aec79d'],
+    }),
+    ('pytoml', '0.1.21', {
+        'checksums': ['8eecf7c8d0adcff3b375b09fe403407aa9b645c499e5ab8cac670ac4a35f61e7'],
+    }),
+    ('regex', '2021.3.17', {
+        'checksums': ['4b8a1fb724904139149a43e172850f35aa6ea97fb0545244dc0b805e0154ed68'],
+    }),
+    ('intreehooks', '1.0', {
+        'checksums': ['87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1'],
+    }),
+    ('pylev', '1.4.0', {
+        'checksums': ['9e77e941042ad3a4cc305dcdf2b2dec1aec2fbe3dd9015d2698ad02b173006d1'],
+    }),
+    ('pastel', '0.2.1', {
+        'source_tmpl': SOURCE_WHL,
+        'checksums': ['4349225fcdf6c2bb34d483e523475de5bb04a5c10ef711263452cb37d7dd4364'],
+    }),
+    ('clikit', '0.6.2', {
+        'source_tmpl': SOURCE_WHL,
+        'checksums': ['71268e074e68082306e23d7369a7b99f824a0ef926e55ba2665e911f7208489e'],
+    }),
+    ('entrypoints', '0.3', {
+        'source_tmpl': SOURCE_WHL,
+        'checksums': ['589f874b313739ad35be6e0cd7efde2a4e9b6fea91edcc34e58ecbb8dbe56d19'],
+    }),
+    ('SecretStorage', '2.3.1', {
+        'modulename': False,  # Doesn't work properly under Python2, but is required for keyring
+        'checksums': ['3af65c87765323e6f64c83575b05393f9e003431959c9395d1791d51497f29b6'],
+    }),
+    ('keyring', '18.0.1', {
+        'modulename': False,  # Doesn't work properly if HOME directory contains keys
+        'checksums': ['67d6cc0132bd77922725fae9f18366bb314fd8f95ff4d323a4df41890a96a838'],
+    }),
+    ('keyrings.alt', '3.2.0', {
+        'modulename': False,  # Doesn't work properly if HOME directory contains keys
+        'checksums': ['1c9981c351dabe902172ccf75bccff78185548f15ad51d5297e6366c0f4c3b51'],
+    }),
+    ('tomlkit', '0.7.0', {
```

Comment:
according to pypi 0.7.2 is the latest for py2.7 which is what py3 for 11.2.0 uses